api/scripts/send-invitations-to-sco-organizations.js

```diff
@@ -12,6 +12,8 @@ const organizationInvitationService = require('../lib/domain/services/organizati
 const organizationRepository = require('../lib/infrastructure/repositories/organization-repository');
 const organizationInvitationRepository = require('../lib/infrastructure/repositories/organization-invitation-repository');
 
+const TAGS = ['JOIN_ORGA'];
```

Comment:
```suggestion
const TAGS = ['ORGA_INVITATION'];
```
En fait, il y a ces deux notions (join / invitate), ce serait bien d'en choisir une des deux pour ne pas être ambigü